dhis-2/dhis-services/dhis-service-administration/src/main/java/org/hisp/dhis/resourcetable/table/CategoryOptionComboNameResourceTable.java

```diff
@@ -63,10 +66,13 @@ public ResourceTableType getTableType()
     @Override
     public String getCreateTempTableStatement()
     {
-        return "create table " + getTempTableName() +
-            " (categoryoptioncomboid bigint not null primary key, " +
-            "categoryoptioncomboname varchar(255), approvallevel integer, " +
-            "startdate date, enddate date)";
+        StringBuilder sql = new StringBuilder();
```

Comment:
I am not quite sure if this makes the code more readable. I would actually prefer the way it was before. Internally, the JVM will optimize string memory usage so there is not much to gain from a memory perspective either.

Reply:
Done